graspologic/plot/plot.py

```diff
@@ -671,6 +674,248 @@ def pairplot(
     return pairs
 
 
+def _plot_ellipse(
```

Comment:
_plot_ellipse_and_data

Reply:
please leave a reference here to the sklearn code/tutorials that it came from

Reply:
https://github.com/microsoft/graspologic/pull/541/commits/f5217d32c9406544182b2f42a895b172a77b0449 for the citation

Reply:
https://github.com/microsoft/graspologic/pull/541/commits/14e7df5b98934abd92329ff710afa3b08bf62c73 change to rec function name

Reply:
sklearn, not matplotlib: https://scikit-learn.org/stable/auto_examples/mixture/plot_gmm_covariances.html#sphx-glr-auto-examples-mixture-plot-gmm-covariances-py

Reply:
https://github.com/microsoft/graspologic/pull/541/commits/3256949837bc1b38a8e8d3da876c166f39193c7b